melodic/distribution.yaml

```diff
@@ -3530,6 +3530,16 @@ repositories:
       url: https://github.com/ros/genpy.git
       version: kinetic-devel
     status: maintained
+  genpyi:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/rospypi/genpyi-release.git
+      version: 0.2.0-1
+    source:
+      type: git
+      url: https://github.com/rospypi/genpyi.git
```

Comment:
I have to say that I am not a fan of the name of this package.  It could very easily be confused with `genpy`, which is a core ROS package.

What does this package do?

Reply:
This package generates `.pyi` files for ROS message/service files.
`.pyi` is [a Python stub file](https://www.python.org/dev/peps/pep-0484/#stub-files) and mainly used for static type checking tool, like mypy.
Since this package depends `genmsg` and provides the cmake file to extend its supported languages, I followed the naming rule of other packages, like genpy, gencpp.
https://github.com/rospypi/genpyi

But it might be confusing and I don't want to cause inconvenience to ROS users, please let me change this package name if needed. How about `genpystub`? Does it sound better?

Reply:
I would like to update the package name to `genmypy` in order to avoid confusion with `genpy`.
But before that, It would be really appreciated if I could have your feedback on that name. Is the package name `genmypy` acceptable for ros-distro?